jsk_fetch_robot/README.md

```diff
@@ -252,4 +252,79 @@ ssh fetch15
 sudo bash
 rosrun ps3joy sixpair
 rosrun ps3joy ps3joy.py  # with pushing the center button of the joystick
+```
+
+  You might be forget to `source setup.bash` before you run `roseus`
+
+Administration
+--------------
+
+- set global environment variables https://github.com/jsk-ros-pkg/jsk_robot/issues/859#issuecomment-341269420
+```
+$ cat /etc/profile.d/jsk.sh 
+# added by furushchev (2017.11.2)
+export ROSLAUNCH_SSH_UNKNOWN=1                 # enable to run roslaunch with " Server' not found in known_hosts" 
+export JSK_DATA_CACHE_DIR=/etc/ros/jsk_data    # store recognition data within common directories to reduce hdd usage
+```
+
+**NOTE**  
+Scripts located in `/etc/profile.d` will be enabled on next login to the shell, so it is necessary to first log out and re-login to apply this change to current users.
+
+- Create directory for cache data for JSK repository
+
+```bash
+$ sudo mkdir /etc/ros/jsk_data && sudo chmod 0777 /etc/ros/jsk_data
+```
+
+- change permissoin of log direcotry https://github.com/jsk-ros-pkg/jsk_robot/issues/859#issuecomment-341269420
+
+`logrotate` does not work correctly due to directory permission
+```
+$ sudo /usr/sbin/logrotate /etc/logrotate.d/ros
+error: skipping "/var/log/ros/fd645e8c-9a09-11e5-8547-d8cb8a40210c/head_camera-depth_registered_rectify_depth-12-stdout.log" because parent directory has insecure permissions (It's world writable or writable by group which is not "root") Set "su" directive in config file to tell logrotate which user/group should be used for rotation.
+```
+Changed `/var/log/ros` manually
+```
+sudo chmod g-w /var/log/ros
+(cd /var/log/ros && find -type d | xargs sudo chmod g-w)
+```
+c.f.
+```
+furushchev@fetch15:/var/log/ros$ ls -lFahd
+drwxrwsr-x 381 ros ros 36K Nov  2 01:45 ./
+furushchev@pr2:/var/log/ros$ ls -lFahd
+drwxr-xr-x 6 ros ros 36K 11月  1 15:25 ./
+```
+
+- change script for auto `undocking` to disable auto rotatation after unplugged
+
+```diff
+# /opt/ros/indigo/lib/fetch_auto_dock/undock_on_button.py
+44c44,47
+<         goal.rotate_in_place = True
+---
+>         # fixed by furushchev (2017/11/1)
+>         # Disabled rotate in place feature
+>         # goal.rotate_in_place = True
+>         goal.rotate_in_place = False
+```
+
+- Set branch of `jsk_demos` as `master` in order to update jsk_maps
+
+- Add `pr2eus` package to `/home/fetch/ros/indigo/src/jsk-ros-pkg` to use the newest `:speak` function
+
+- Set `amcl/odom_alpha*` paramters manually in `/opt/ros/indigo/share/fetch_navigation/launch/include/amcl.launch`, to prevent jumps of self-position of fetch
+
+- Fix wifi access point to catch access point with strong radio wave intensity
+
+- To add `cartesian_wrench_controller` to `default controller`,
+
+```
+source /home/fetch/fetch_controller_ws/devel/setup.bash # in /home/fetch/ros/indigo_robot/devel/setup.bash
+```
+
+- To use `respeaker` and `julius_ros`
```

Comment:
add PR and release version

Reply:
We use `julius_ros` of `2.1.10` of `jsk_3rdparty`.

@furushchev 
Could you please create PR for your `respeaker_ros` package?
If you do not have time to create the PR, can I create the PR?